### DISCUSSION TITLE
Using Relay client with Redis Sentinel

### DISCUSSION THREAD

**Discussion by nickchomey** [rhubarbgroup/.github#27] [Support (ROC)] [answered]
The readme shows the following for using ROC with Redis Sentinel

```
define( 'WP_REDIS_CLIENT', 'predis' );

define( 'WP_REDIS_SENTINEL', 'my-sentinel' );
define( 'WP_REDIS_SERVERS', [
    'tcp://127.0.0.1:5380',
    'tcp://127.0.0.2:5381',
    'tcp://127.0.0.3:5382',
] );
```

Is it possible to use Relay client with Redis Sentinel, or just predis?

**Comment by tillkruss** [comment, marked as answer]
Only with Object Cache Pro and not planned anytime soon for Redis Object Cache. Pull requests welcome.

**Reply by nickchomey** [reply]
Thanks for the confirmation! In the coming months I'll should be able to put some attention towards this and dig into predis, credis, phpredis etc and see how it all works. I'll provide a pr if I'm able.
